marcopeg/demo-watski
# Can now view a list of posts on the home page.

# Benefits added
Now possible to view a list of all posts on the home page. 

# Testing instructions
Go to the homepage in the browser and verify that a table with 3 posts are rendered there. Each of them should have a post-title and and author name. 

# Issues related
closes #2
